network/builder/src/dummy.rs

```diff
@@ -154,6 +154,7 @@ pub fn setup_network() -> DummyNetwork {
         listener_addr,
         authentication_mode,
         constants::MAX_FRAME_SIZE,
+        false,
```

Comment:
let's add block comments like `false, /* enable_proxy_protocol */` to aid readability

Reply:
Done